python/ql/test/library-tests/regex/Qualified.ql

```diff
@@ -2,5 +2,6 @@ import python
 import semmle.python.regex
 
 from Regex r, int start, int end, boolean maybe_empty
-where r.qualifiedItem(start, end, maybe_empty)
+where
+  r.qualifiedItem(start, end, maybe_empty) and r.getLocation().getFile().getShortName() = "test.py"
```

Comment:
Do you want this change or should we rather not extract dependencies?

Reply:
Restricting the import depth is probably the better choice, yeah.

Reply:
Done.